contribs/gmf/src/datasource/Manager.js

```diff
@@ -243,6 +243,21 @@ export class DatasourceManager {
     this.ngeoDataSources_.map = map;
   }
 
+  /**
+   * Get a datasource by its name.
+   * @param {string} name The name of the datasource.
+   * @hidden
+   */
+  getDatasource(name) {
+    for (const datasourceId in this.dataSourcesCache_) {
+      if (Object.prototype.hasOwnProperty.call(this.dataSourcesCache_, datasourceId)) {
```

Comment:
```suggestion
      if (this.dataSourcesCache_.hasOwnProperty(datasourceId) {
```
It is not the same in shorter ?